ksql-engine/src/main/java/io/confluent/ksql/security/KsqlAuthorizationValidatorFactory.java

```diff
@@ -61,6 +61,18 @@ private KsqlAuthorizationValidatorFactory() {
     return Optional.empty();
   }
 
+  private static KsqlAuthorizationValidator createAuthorizationValidator(KsqlConfig ksqlConfig) {
+    KsqlAccessValidator accessValidator = new KsqlAccessValidatorProvider();
+
+    // The cache expiry time is used to decided whether to enable the cache or not
+    long expiryTime = ksqlConfig.getLong(KsqlConfig.KSQL_AUTH_CACHE_EXPIRY_TIME);
```

Comment:
to self: Guess it's fine to just check the expiryTime and not the max.entries  config, since our main goal here is to refresh every so often..

Reply:
I think so. I was thinking of using another variable, though, to enable cache, like `ksql.auth.cache.enable`, but I wasn't sure about it. what do you think?

Reply:
Almost always, you don't want to disable this cache.. So having `0` as a special sentinel is ok IMO. prevents adding yet another config..